app/models/cracked_ineffective_trial.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+class CrackedIneffectiveTrial
+  include ActiveModel::Model
+
+  attr_accessor :body
+
+  def id
+    body["id"]
```

Comment:
Would there be any sensible clarifying nouns to goes with these values, similar to in the offence model? https://github.com/ministryofjustice/laa-court-data-adaptor/blob/master/app/models/offence.rb